support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -138,12 +137,6 @@ const paymentFrequencyButtonsCss = css`
 	}
 `;
 
-const tabletLineBreak = css`
-	${from.desktop} {
-		display: none;
```

Comment:
The need for this css is replaced by the `text-wrap: balance` on the heading. It means the line break still happens after the comma when it wraps:
![Screenshot 2025-02-04 at 10 04 22](https://github.com/user-attachments/assets/3734f357-69fa-4149-afcc-cb5ce02df4c0)
And it saves us from needing separate config for mobile vs desktop